Composer/packages/client/src/recoilModel/undo/undoHistory.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Snapshot } from 'recoil';
+
+export class UndoHistory {
+  public stack: Snapshot[] = [];
+  public present = -1;
+  public initialLocation = { dialogId: '', selected: '', focused: '' };
+
+  public undo() {
+    if (!this.canUndo()) throw new Error('Undo is not support');
```

Comment:
Typo: not supported

Reply:
In addition how does this error get propogated upwards @lei9444 . Like throwing this error does it just show up a message in the ErrorBoundary component we have?

Reply:
I want to use this throw error to avoid other developers to use undoHistory.undo() directly.